src/main/java/com/faforever/client/game/PlayerCardController.java

```diff
@@ -169,6 +171,7 @@ public void openContextMenu(MouseEvent event) {
           .build()
           .show(getRoot().getScene().getWindow(), event.getScreenX(), event.getScreenY());
     }
+    event.consume();
```

Comment:
Why do you consume the event here?

Reply:
Without this line clicking the player name also results in the detail card opening